opensanctions/crawlers/eu_europol_most_wanted.py

```diff
@@ -0,0 +1,94 @@
+import re
+from datetime import datetime
+from unicodedata import normalize
+
+from lxml import html
+
+
+def crawl(context):
+    base_url = context.dataset.data.url  # "https://eumostwanted.eu/"
+    response = context.http.get(base_url)
+    doc = html.fromstring(response.text)
+    # Find all link on page
+    all_links = doc.findall(".//a")
+    # remove all other links
+    most_wanted = [
+        a.get("href") for a in all_links if a.get("href", "").startswith("http")
+    ]
+    # filter out other links so we have only links to most_wanted peoples
+    most_wanted_cleaned = [link.replace(base_url, "") for link in most_wanted]
+    most_wanted_cleaned = [
+        link
+        for link in most_wanted_cleaned
+        if link not in ("legal-notice", "https://www.europol.europa.eu")
+    ]
+    context.log.info(f"Found {len(most_wanted_cleaned)} people on the eu-most-wanted list")
+    # crawl every pages and create a new person
+    for person_id in most_wanted_cleaned:
+        context.log.info(f"crawling {person_id}")
+        crawl_person(context, person_id, base_url)
+
+
+def crawl_person(context,person_id, base_url):
+    """read and parse every person-page"""
+    new_url = base_url + person_id
+    p_response = context.http.get(new_url)
+    p_doc = html.fromstring(p_response.text)
+    xpath_infofield = '//*[contains(concat( " ", @class, " " ), concat( " ", "wanted_top_right", " " ))]'
+    infofields = p_doc.xpath(xpath_infofield)[0]
+    # use values and keys() to filter, there is propably a better way to do
+    # this but at least it workeds.
+    person = context.make("Person")
+    person.id = context.make_slug(person_id)
```

Comment:
I've seen a lot of sources now that re-use the ID in weird and surprising ways against all reason. So I'd add the name in here if doable, just to avoid these sorts of clashes.